docs/docsite/rst/plugins/strategy.rst

```diff
@@ -34,7 +34,7 @@ or in the `ansible.cfg` file:
     [defaults]
     strategy=linear
 
-You can also specify the strategy plugin in the play via the :ref:`strategy` keyword::
+You can also specify the strategy plugin in the play via the ``strategy`` :ref:`keyword <playbooks_keywords>` in a play::
```

Comment:
I'd make the link label be "strategy keyword".

```
:ref:`strategy keyword <playbook_keywords>`
```

To be even more effective, we'd want to add targets for each keyword, though....

Reply:
Yeah, there's a lot of optimization that could be done beyond fixing the links. Maybe we should open issues for these kinds of ideas? 

This particular link still triggers an error, which is making me a little crazy.

Reply:
Label changed.

Reply:
Yeah, issues if that's how you want to track them.  I don't know what your workflow for a todo list is.